### PR TITLE
updated christmas tree report dates

### PR DESCRIPTION
**Pull request by mtlaney**
## Summary
Fixes report query dates to correctly display data received from the server

## Impacted Areas of the Site
Admin christmas trees reports when searching by date

## This pull request changes...
- [ ] Correctly display query dates entered into the search parameters
- [ ] Correctly displays data when querying for single day parameters

## This pull request is ready to merge when...
- [x] Feature branch is starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [ ] Tests have been updated 
- [ ] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [x] Server actions captured by logs (manual)
- [x] Documentation / readme.md updated (manual)
- [x] API docs updated if need (manual)
- [x] JSDocs updated (manual)
- [x] Docker updated if needed (manual)
- [ ] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: server/src/controllers/christmas-tree/admin.es6 [hunk 1]
```diff
@@ -28,9 +28,7 @@ const getPermitResult = (permit) => {
   if (permit.christmasTreesForest && permit.christmasTreesForest.timezone) {
     eachPermit.issueDate = moment.tz(permit.updatedAt, permit.christmasTreesForest.timezone).format('MM/DD/YYYY');
 
-    eachPermit.expireDate = moment
-      .tz(permit.permitExpireDate, permit.christmasTreesForest.timezone)
-      .format('MM/DD/YYYY');
+    eachPermit.expireDate = moment.tz(permit.permitExpireDate, permit.christmasTreesForest.timezone).format('MM/DD/YYYY');
```
Comment (davemcorwin) [inline server/src/controllers/christmas-tree/admin.es6]: If there are files that ONLY have formatting changes, can we not include them in the PR to reduce the noise a bit?

Comment (mtlaney) [reply server/src/controllers/christmas-tree/admin.es6]: yes, stupid prettier plugin, thought I had admin checked out, reverting / truncating and re-pushing

**Comment by mtlaney** [comment]
@davemcorwin 
Should be good to go now. Removing the forest time zones since they weren't needed on date only queries led down a rabbit hole. Date-time-range form fields searching for date-only for some reason had default start time as 0100 and default end time at 0200 so it wasn't capturing anything that took place outside of that hour.

**Comment by davemcorwin** [comment]
My general thought is that there is no reason the UI should do anything with timezones. The assumption for the user is that whatever date and time they see, it is in the timezone of the relevant forest.

To enable this, the backend should have responsibility for providing all dates and times in the timezone of the forest with no need to provide the timezone information itself. I believe this is currently happening. The datetimes come out of the database as UTC, are converted to forest timezone, formatted to just date and time, and sent to the client. On the flip side, when dates and times are received from the UI, the server should convert them to timezone aware datetimes with the assumption they are in the timezone of the relevant forest.

**Comment by davemcorwin** [comment]
Per our discussion, @mtlaney is going to look at ripping out timezone logic in the UI so it can assume all dates and times are in forest timezone. @davemcorwin will verify, test and make any changes on the backend.

**Comment by mtlaney** [comment]
re-adding times now, district - dates is using them

**Comment by mtlaney** [comment]
Digging into e2e now

**Comment by mtlaney** [comment]
Alright, after removing tz refs, and utc refs, we're now failing e2e's due to the regex written into util.es6, which is expecting utc formatted dates from the client. 

It appears to be an issue with conversion between server and client.

tempoutfitter-application.es6
`    tempOutfitterFieldsActDescFieldsStartDateTime: {
      type: Sequelize.STRING(255),
      field: 'temp_out_act_desc_serv_strt_dt',
      validate: {
        len: {
          args: [0, 255],
          msg: 'tempOutfitterFieldsActDescFieldsStartDateTime must be less than 255 characters in length'
        },
        isValidDateTime(value) {
          if (!util.validateDateTime(value)) {
            throw new Error('tempOutfitterFieldsActDescFieldsStartDateTime must be a valid UTC string');
          }
        }
      }
    },`

temp-outfitter.es6
`  output.tempOutfitterFieldsActDescFieldsStartDateTime = input
    .tempOutfitterFields.activityDescriptionFields.dateTimeRange.startDateTime;`

@davemcorwin lets roundup when you can and we'll figure out the best way to go forward.

File: frontend/src/app/trees/_services/christmas-trees-application.service.ts [hunk 2]
```diff
@@ -111,18 +111,18 @@ export class ChristmasTreesApplicationService {
   /**
    * @returns Update distrct dates for forest district
    */
+
+  //  double check before commit, should be good though
   updateDistrictDates(forest, districtName, startDate, endDate) {
     const cuttingAreas = Object.assign({}, forest.cuttingAreas);
-    const format = 'YYYY-MM-DDTHH:mm:ss';
+    const format = 'MM-DD-YYYY';
```
Comment (davemcorwin) [inline frontend/src/app/trees/_services/christmas-trees-application.service.ts]: we don't need the time here anymore?

Comment (mtlaney) [reply frontend/src/app/trees/_services/christmas-trees-application.service.ts]: I'll double check quick

**Pull request merged**